README.md

```diff
@@ -55,7 +54,6 @@ A full list supported options are listed here.
 |:----------------------:|:----------------------:|------------------------------------------------------------------------------------|:--------:|:----------------:|
 |          `app`         |           -a           | The path to the host application to execute (your .app)                            |     Y    | n/a              |
 |      `output-dir`      |           -o           | Directory where to put output log files (bluepill only)                            |     Y    | n/a              |
-|      `scheme-path`     |           -s           | The scheme to run tests                                                            |     Y    | n/a              |
```

Comment:
Can you add an entry for `xctestrun-path` to this table? That is what I looked at first when first starting.